apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -223,6 +240,10 @@ const styles = {
     display: 'flex',
     textAlign: 'center',
   },
+  sortOrderSelect: {
+    marginRight: 10,
+    width: parseInt(styleConstants.STUDENT_LIST_WIDTH) - 10,
```

Comment:
I don't *think* these two numbers are related, but if they are can they be a constant in case one is ever changed?

Reply:
They very much are related. To put the "Lessons attempted in..." text in the right spot the width of this select dropdown needs to be (student list width - margin). I'll put in a constant for this.